src/main/java/rx/Completable.java

```diff
@@ -1927,7 +1933,12 @@ public void onSubscribe(Subscription d) {
         
         return mad;
     }
-    
+
+    private static void deliverUncaughtException(Throwable e) {
```

Comment:
That would require it to be public, sadly, and this isn't an API that RxJava needs to provide for users.